subprojects/logging/src/integTest/groovy/org/gradle/internal/logging/ConsoleJvmTestLoggingFunctionalTest.groovy

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.internal.logging
+
+import org.gradle.integtests.fixtures.AbstractConsoleFunctionalSpec
+import org.gradle.integtests.fixtures.executer.ExecutionResult
+
+class ConsoleJvmTestLoggingFunctionalTest extends AbstractConsoleFunctionalSpec {
+
+    private static final String TEST_TASK_NAME = 'test'
+    private static final String TEST_TASK_PATH = ":$TEST_TASK_NAME"
+    private static final String JAVA_TEST_FILE_PATH = 'src/test/java/MyTest.java'
+
+    def setup() {
+        buildFile << javaProject()
+    }
+
+    def "can group failed test log event with task by default"() {
+        executer.withStackTraceChecksDisabled()
```

Comment:
I don't think this should be necessary, so there's something wrong with our fixtures or there's something wrong with the way we handle failures with the new console output (or both).